src/hooks/useAwsRoute.ts

```diff
@@ -50,7 +50,7 @@ const useAwsRoute = () => {
 					record([{ EventType: EventTypeEnum.ROUTE_SEARCH, Attributes: recordAttributes }]);
 				}
 			},
-			setRoutePositions: (p: Position | undefined, type: InputType) => {
+			setRoutePositions: (p: number[] | undefined, type: InputType) => {
```

Comment:
Is there a specific reason that Position has been changed to number[] everywhere?

Reply:
Since we have updated to V3 of aws-sdk
`import { CalculateRouteRequest } from "@aws-sdk/client-location";`
doesn't provide an export for Position, which is why that has been updated.